src/pages/workspace/WorkspacePeoplePage.js

```diff
@@ -0,0 +1,328 @@
+import Text from '../../components/Text';
+import ROUTES from '../../ROUTES';
+import ConfirmModal from '../../components/ConfirmModal';
+import personalDetailsPropType from '../personalDetailsPropType';
+import withWindowDimensions, {windowDimensionsPropTypes} from '../../components/withWindowDimensions';
+
+const propTypes = {
+    ...withLocalizePropTypes,
+
+    ...windowDimensionsPropTypes,
+
+    /** The personal details of the person who is logged in */
+    personalDetails: personalDetailsPropType.isRequired,
+
+    /** The policy passed via the route */
+    policy: PropTypes.shape({
+        /** The policy name */
+        name: PropTypes.string,
+    }),
+
+    /** URL Route params */
+    route: PropTypes.shape({
+        /** Params from the URL path */
+        params: PropTypes.shape({
+            /** policyID passed via route: /workspace/:policyID/people */
+            policyID: PropTypes.string,
+        }),
+    }).isRequired,
+};
+
+const defaultProps = {
+    policy: {
+        name: '',
+    },
+};
+
+class WorkspacePeoplePage extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            selectedEmployees: [],
+            isRemoveMembersConfirmModalVisible: false,
+        };
+
+        this.inviteUser = this.inviteUser.bind(this);
+        this.removeUsers = this.removeUsers.bind(this);
+        this.toggleUser = this.toggleUser.bind(this);
+        this.renderItem = this.renderItem.bind(this);
+        this.renderHeader = this.renderHeader.bind(this);
+        this.askForConfirmationToRemove = this.askForConfirmationToRemove.bind(this);
+        this.hideConfirmModal = this.hideConfirmModal.bind(this);
+    }
+
+    /**
+    * Open the modal to invite a user
+    */
+    inviteUser() {
+        Navigation.navigate(ROUTES.getWorkspaceInviteRoute(this.props.route.params.policyID));
+    }
+
+    /**
+    * Remove selected users from the workspace
+    */
+    removeUsers() {
+        // Remove the admin from the list
+        // eslint-disable-next-line max-len
+        removeMembers(this.state.selectedEmployees.filter(email => email !== this.props.session.email), this.props.route.params.policyID);
+        this.setState(prevState => ({
+            ...prevState,
+            selectedEmployees: [],
+            isRemoveMembersConfirmModalVisible: false,
+        }));
+    }
+
+    /**
+    * Show the modal to confirm removal of the selected members
+    */
+    askForConfirmationToRemove() {
+        this.setState(prevState => ({...prevState, isRemoveMembersConfirmModalVisible: true}));
+    }
+
+    /**
+    * Hide the confirmation modal
+    */
+    hideConfirmModal() {
+        this.setState(prevState => ({...prevState, isRemoveMembersConfirmModalVisible: false}));
+    }
+
+    /**
+    * Add or remove user from the selectedEmployees list
+    *
+    * @param {Sting} login
+    */
+    toggleUser(login) {
+        if (login === 'ALL') {
```

Comment:
I think this might be clearer as a separate method `toggleAllUsers()` this way we don't need to introduce this constant or special significance for the `ALL` user and it will be very clear what will happen.

`toggleUser('ALL')` vs `toggleAllUsers() + toggleUser('some user')`

Reply:
Ah missed this one. Yes, I agree, that will be clearer. I will refactor it.